example/scalalib/native/1-simple/build.mill

```diff
@@ -0,0 +1,56 @@
+//// SNIPPET:BUILD
+package build
+import mill._, scalalib._, scalanativelib._
+
+object `package` extends RootModule with ScalaNativeModule {
+  def scalaVersion = "2.13.11"
+  def scalaNativeVersion = "0.5.5"
+
+  // You can have arbitrary numbers of third-party dependencies
+  def ivyDeps = Agg(
+    ivy"com.lihaoyi::mainargs::0.7.6"
+  )
+
+  object test extends ScalaNativeTests {
+    def ivyDeps = Agg(ivy"com.lihaoyi::utest::0.8.4")
+    def testFramework = "utest.runner.Framework"
+  }
+}
+//// SNIPPET:END
+//
+//
+//// SNIPPET:DEPENDENCIES
+//
+//
+/** Usage
+
+
+> ./mill nativeLink  # Build and link native binary
+...
+compiling 1 Scala source to ...
+
+> ./mill releaseMode # Generate some files in `out` directory about the `releaseMode`. Default releaseMode is set to Debug.
+
+> ./mill nativeOptimize # Generate some files in `out` directory about native optimization. The Default value is true.
+
+> ./mill nativeIncrementalCompilation # Generate config files in `out` directory about native incremental compilation. Default is false.  
+
+> ./mill nativeLinkingOptions # Generate config files in `out` directory about the path in native linking options. Default path is `"-L/usr/local/lib`.
```

Comment:
The comments here are all rubbish. What are you trying to achieve here? Let's just skip these since they'll be covered in common-config anyway